### ISSUE TITLE
"Comment Section"

### ISSUE THREAD

**Issue by mastery3** [mastery3/sharpness6#1]
Since Modrinth does not have a comment section, here you go.

**Comment by bbauti** [comment]
Hello! is there a way to make the Fortune enchant maximun level greater than 3?

**Comment by mastery3** [comment]
> Hello! is there a way to make the Fortune enchant maximun level greater than 3?

Not really, although I'll implement it next time I get around to updating the mod.

**Comment by KKolyanGGWP** [comment]
is there any way to make villagers sell new enchants? Or for example for wandering trader. Cuz i wanted to make rare echants in Wandering Trades

**Comment by KKolyanGGWP** [comment]
Also, maybe its easier to make discord server? Its much easier for me to just join a discord server

**Comment by Cam396** [comment]
There is a way to get higher level enchants, I added support for almost all enchants that scale with levels in my latest pull request

**Comment by K0KE11** [comment]
This is like the mod https://www.curseforge.com/minecraft/mc-mods/max-enchant-x but at least it seems this one is updateed more frequently which I'm grateful for so I just wanna say thank you!

**Comment by nomnom597** [comment]
Do I just open the json with notepad and change any number to whatever maximum I wish? I set Power to 10, and when I tried to anvil a book with punch 1 and power 2 into a bow, I only got punch. No power. Am I screwing this up somehow?

**Comment by mastery3** [comment]
@nomnom597 Either a config syntax error or a mod incompatibility. Put your config file here as an attachment.

**Comment by AXCWG** [comment]
1.21, why does sharpness X dont seems to be working? the damage is not being higher...

**Comment by nomnom597** [comment]
> @nomnom597 Either a config syntax error or a mod incompatibility. Put your config file here as an attachment.

seems to work from installation forwards, just didn't like my existing book. and AXCWG, I didn't realize how LITTLE sharpness X helps, every 10 levels of sharpness only adds 2.5 hearts of damage. After I realized that I went straight to 30 lol

**Comment by AXCWG** [comment]
> > @nomnom597 Either a config syntax error or a mod incompatibility. Put your config file here as an attachment.
> 
> seems to work from installation forwards, just didn't like my existing book. and AXCWG, I didn't realize how LITTLE sharpness X helps, every 10 levels of sharpness only adds 2.5 hearts of damage. After I realized that I went straight to 30 lol

😂kinda just realized after going way higher enchanting levels
